managed/ui/src/components/config/Storage/EditBackupList.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) YugaByte, Inc.
+//
+// Author: Nishant Sharma(nishant.sharma@hashedin.com)
+//
+// This file will hold a editable view for the particular storage
+// configuration input fields.
+
+import { Field } from 'redux-form';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+
+/**
+ * This method is used to handle the edit backup storage
+ * config form.
+ *
+ * @param {object} data Respective row details.
+ * @param {string} fieldId Input field id.
+ * @param {string} configName Input field name.
+ * @returns inUse ? true : false;
+ */
+const disableFields = (data, fieldId, configName) => {
+  if (data.inUse && fieldId !== `${configName}_CONFIGURATION_NAME`) {
+    return true;
+  }
+};
```

Comment:
Prefer this to be set as a variable. Just create another `const` in EditBackupList.

Reply:
Done.